modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,15 @@
+== Inline formatting.
+
+List bullets and list numbers can be spot- or inline-formatted.
+
+If an end-user selects the contents of a list item and applies inline formatting, such as a change of color or change of font-size, this spot formatting is also applied to the list item’s associated bullet or number.
```

Comment:
We might want to make it clear here that it's the **entire contents** of a list item, because selecting partial contents shouldn't change the bullet.

Reply:
Edited to make this explicit.